tests/psalm-baseline.xml

```diff
@@ -39,4 +137,10 @@
       <code>$transport</code>
     </InvalidArgument>
   </file>
+  <file src="vendor/cerdic/css-tidy/class.csstidy.php">
+    <InvalidDocblock occurrences="2">
+      <code>public function parse_string_list($value) {</code>
```

Comment:
unsure why psalm report this, but I sent a PR in any case https://github.com/Cerdic/CSSTidy/pull/61